test/Nancy.Tests/Unit/Bootstrapper/NancyBootstrapperBaseFixture.cs

```diff
@@ -131,17 +131,11 @@ public void Should_invoke_startup_tasks_after_registration_tasks()
             this.bootstrapper.OverriddenRegistrationTasks = new[] { registrations };
 
             // When
-            using(var scope = Fake.CreateScope())
-            {
-                this.bootstrapper.Initialise();
-
-                // Then
-                using (scope.OrderedAssertions())
-                {
-                    A.CallTo(() => registrations.CollectionTypeRegistrations).MustHaveHappened();
-                    A.CallTo(() => startup.Initialize(A<IPipelines>._)).MustHaveHappened();
-                }
-            }
+            this.bootstrapper.Initialise();
+
+            // Then
+            A.CallTo(() => registrations.CollectionTypeRegistrations).MustHaveHappened()
+                .Then(A.CallTo(() => startup.Initialize(A<IPipelines>._)).MustHaveHappened());

```

Comment:
[Ordered assertions](http://fakeiteasy.readthedocs.io/en/2.0.0/ordered-assertions/) have been completely overhauled in FIE 2.0. Scopes no longer exist.